bin/yarn

```diff
@@ -1,11 +1,17 @@
 #!/usr/bin/env ruby
+require 'pathname'
```

Comment:
Out of curiosity - was this boilerplate?

Reply:
Boilerplate: https://github.com/rails/rails/blob/master/railties/lib/rails/generators/rails/app/templates/bin/yarn.tt